ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -542,7 +542,7 @@ tests:
     workflow: openshift-e2e-nutanix-sdn
 - as: e2e-agent-compact-ipv4
   cluster: build05
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg/.*|terraform/providers)/(alibabacloud|aws|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  run_if_changed: agent
```

Comment:
#34878 is making the same change, plus deleting the e2e-agent-mce job, so I think we can drop these changes from this patch and merge both to avoid conflicts.

Reply:
fixed